merino-settings/src/lib.rs

```diff
@@ -84,6 +84,9 @@ pub struct Settings {
 
     /// Settings to use when determining the location associated with requests.
     pub location: LocationSettings,
+
+    /// A blocklist to filter suggestions coming providers. Optional.
+    pub suggestion_blocklist: Option<HashMap<String, String>>,
```

Comment:
I don't think we should make this optional. This provider doesn't make sense without a blocklist.

Reply:
Whoops, this is actually bogus code: the blocklist is part of the provider configuration, not the general configuration.